src/lucky_flow/driver.cr

```diff
@@ -1,7 +1,24 @@
 abstract class LuckyFlow::Driver
   @retry_limit : Time = 2.seconds.from_now
 
+  getter session : Selenium::Session { start_session }
+
   abstract def start_session : Selenium::Session
+  abstract def stop
+
+  def reset : Nil
+    @session.try &.cookie_manager.delete_all_cookies
```

Comment:
we use `@session` instead of the getter so that it doesn't start a session if there wasn't one already

Reply:
It will be nil, so the `.try` will avoid an error